README.md

```diff
@@ -13,6 +13,8 @@ Before contributing, please review the [Contribution Guidelines](https://github.
 
 - [CMS Acceptable Risk Safeguards](https://github.com/CMSgov/ars-machine-readable): the tailored profiles and catalog of adapted NIST SP 800-53 controls used by the Centers for Medicare and Medicaid Services in OSCAL format. Perhaps the first OSCAL content released by a US government agency other than NIST, separate of collaboration with FedRAMP.
 
+- [Fathom5 SP 800-171 Catalog](https://github.com/FATHOM5/oscal/tree/main/content/SP800-171/oscal-content): the community-maintained version(s) of the NIST SP 800-171 catalog created by Fathom5.
```

Comment:
Suggestion: Fathom5's OSCAL version  of NIST SP 800-171  Catalog

Reply:
I also wonder if the description statement is endorsed by the FATHOM5 team. Is this catalog maintain by them, or by the community?

Reply:
Sorry I didn't respond sooner, hi @iMichaela. I merged this longer before you gave the PR feedback. Apologies!

If you are still interested, feel free to open a PR and I will merge in.

> I also wonder if the description statement is endorsed by the FATHOM5 team. Is this catalog maintain by them, or by the community?

I have no idea, this is just a listing of community created (and maintained?) content. Unless stated otherwise, I presume they (Fathom5) made it since it is in their GitHub organization they have documentation and ongoing development for and with it.

Maybe @matt-f5 can elaborate?